.github/workflows/packages_publishing.yml

```
@@ -96,7 +96,15 @@ jobs:
       - name: Build artifacts package
         run: npx ts-node tools/scripts/make-artifacts-package
 
-      - uses: actions/upload-artifact@v4
+      - name: Upload SBOM artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: SBOM
```

```suggestion
          name: sbom
```

Let's name it in lowercase to match another workflow artifact: [sbom](https://github.com/DevExpress/DevExtreme/blob/25_2/.github/workflows/build_all.yml#L75)

---

Or we can make both uppercase 🤔